autosubmit/autosubmit.py

```diff
@@ -727,35 +731,40 @@ def parse_args() -> Tuple[int, Optional[argparse.Namespace]]:
     @staticmethod
     def run_command(args):
         expid = "None"
+
+        if hasattr(args, 'notransitive') and args.notransitive:
+            warnings.warn('notransitive is deprecated and will be removed in a future major release!')
```

Comment:
I used `warnings` to display a deprecation warning if any user tries to pass `--notransitive` :+1: There are some other args that I removed but that are not so general, so the changes are within the `if` blocks below for those commands.